extensions/VideoManager/VideoManager.fs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright 2018 Fabulous contributors. See LICENSE.md for license.
+namespace Fabulous.DynamicViews
+
+[<AutoOpen>]
+module VideoManagerExtension =
+
+    open Fabulous.DynamicViews
+    open MediaManager.Forms
+    open MediaManager.Video
+
+    //// Define keys for the possible attributes
+    let SourceAttribKey = AttributeKey<_> "VideoManager_Source"
+    let VideoAspectAttribKey = AttributeKey<_> "VideoManager_VideoAspect"
+    let ShowControlsAttribKey = AttributeKey<_> "VideoManager_ShowControls"
+
+    type Fabulous.DynamicViews.View with
+        /// Describes a VideoView in the view
+        static member inline VideoView
+            (source: obj, ?videoAspect: VideoAspectMode , ?ShowControls: bool,
```

Comment:
Yes. It's from the MediaManager. But I can't give you an explanation of the reason.

Reply:
Ok. Seems this is because the Play function accepts a bunch of different parameters.
https://github.com/martijn00/XamarinMediaManager/blob/develop/MediaManager/IMediaManager.cs